MVC/application/view/components/loginForm.php

```diff
@@ -0,0 +1,38 @@
+<form action="http://mvc-task.com/userControl/userLogin" method="post">
```

Comment:
It's understandable that you've required this form in another page that's why the html tags are missing, but let's mention that here